### PR TITLE
PPTP-1839 : New Submitted Returns Page

### PR DESCRIPTION
**Pull request by RyanKellyHMRC**
#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added - N/A
 - [ ] Links to dependencies have been included (BE/FE work) - N/A
 - [ ] User Acceptance Tests (UAT) were run locally and have passed - N/A
 - [x] No Accessibility errors/warnings reported by Wave

image

### CODE REVIEW

File: app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala [hunk 1]
```diff
@@ -34,4 +34,9 @@ class BtaEntryPointController @Inject() (
       Redirect(returnRoutes.StartDateReturnController.displayPage())
     }
 
+  def submittedReturns(): Action[AnyContent] =
+    authenticate { implicit request =>
+      Redirect(returnRoutes.SubmittedReturnsController.displayPage())
```
Comment (liampan) [inline app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala:39]: Can the route not just point to this controller? save on the double calls to auth?

Comment (liampan) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala:39]: or can we just redirect without auth

Comment (liampan) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala:39]: Cant remeber how it works but could probs use the Default controller to do this

Comment (liampan) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala:39]: https://www.playframework.com/documentation/2.8.x/api/scala/controllers/Default.html

Comment (RyanKellyHMRC) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala:39]: `DefaultController` looks interesting. I wonder whether you can use the reverse routing. I'll take a quick look...

Comment (RyanKellyHMRC) [reply app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/bta/BtaEntryPointController.scala:36]: Couldn't see an obvious way of doing this.

**Review by liampan — APPROVED** [review]

**Pull request merged**